metaproc-connector-demo/config/variables.yaml

```diff
@@ -1,9 +1,5 @@
-# == Variables ==
-# 
-# You can define here your project Variables.
-# If you want to define/override a Variable for a specific Environment, 
-# add an additional ‘variables.yaml’ file in a subdirectory in the ‘Config’ folder: 
-# '<project>/Config/_<environment>/variables.yaml
-#
 Variables:
-#  myVariable: value
+  metaproc:
+    Url: http://89.145.164.140:8080
+    Username: metadmin
+    Password: "#M3t!adm1n?!"
```

Comment:
public credential?

Reply:
****! this is crucial, accidently checked in! but we will kill this host anyway.I change this credential my friend.